products/colp/metadata.yml

```diff
@@ -33,22 +65,39 @@ columns:
       - ["3", Brooklyn]
       - ["4", Queens]
       - ["5", Staten Island]
+    format_overrides:
+      csv:
+        name: BOROUGH
+      socrata:
+        display_name: BOROUGH
```

Comment:
these seem like a mix of format and distribution overrides. if it were format only I think it'd be `csv:` and `shapefile:`

definitely seems reasonable to be thinking about destinations though so I'm not sure which would be a better way to frame these

Reply:
per pairing: reworking structure of overrides